django/contrib/postgres/fields/array.py

```diff
@@ -295,3 +296,11 @@ def __init__(self, start, end):
 
     def __call__(self, *args, **kwargs):
         return SliceTransform(self.start, self.end, *args, **kwargs)
+
+
+@CombinedExpression.dispatch(CombinedExpression.ADD, ArrayField, ArrayField)
+class ArrayCatPair(Func):
+    function = 'ARRAY_CAT'
+
+    def __init__(self, lhs, connector, rhs, output_field):
+        super(ArrayCatPair, self).__init__(lhs, rhs, output_field=output_field)
```

Comment:
You can rebase your branch and target it for Django 2.0. Since master no longer supports Python 2, you can make a few updates such as using `super().`.